staff/clara-rufi/e-commerce-template/e-commerce.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>E-commerce-best-world-bikes</title>
+    <link rel="stylesheet" type="text/css" href="ecommerce.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
+
+
+</head>
+
+    <header>
+      
+        <div class="search-container">
+            <form>
+                <i class="fas fa-bicycle bike"></i>
+                <a class="titleBrand">BWB</a>
+                <input type="text" class="searchBar" placeholder=" Search.." name="search">
+                <button type="submit"><i class="fa fa-search search"></i></button>
+               
+            </form>
+        </div>
+        
+           <div class= "rightnav">
+            <a href="#contact">Contact</a>
+            <a href="#about">About</a>
+            <i class="fas fa-bars hamburguer"></i>
+        </div>  
+     
+    </header>
+
+    <main>
+        <h1>Bike selection</h1>
```

Comment:
Maybe I would put the title h1 inside the section

Reply:
ok, thanks for this suggestion.